build/releasing/index.ts

```diff
@@ -6,11 +6,8 @@ export const config: BeachballConfig = {
     groups: [
         {
             name: "Microsoft FAST",
-            include: [
-                "packages/tooling/*",
-                "packages/utilities/*",
-                "packages/web-components/*",
-            ],
+            include: ["packages/utilities/*", "packages/web-components/*"],
+            exclude: ["packages/tooling/*"],
```

Comment:
Do we want to exclude all tooling from versioning together? I think we just want to exclude the version which is explicitly pinned to `latest`.

```suggestion
            include: ["packages/tooling/*", "packages/utilities/*", "packages/web-components/*",],
            exclude: ["packages/tooling/fast-tooling"],
```

Reply:
@janechu can advise, but I think we just want this to exclude that single package